pkg/handlers/generic/lifecycle/cni/calico/strategy_helmaddon.go

```diff
@@ -102,7 +102,7 @@ func (s helmAddonStrategy) apply(
 			ValuesTemplate:   values,
 		},
 	}
-
```

Comment:
nit: please keep the spacing consistent either with a newline with these changes across the different files.

Reply:
the spacing is consistent.